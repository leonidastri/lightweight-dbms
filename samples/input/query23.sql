select distinct B2.E, S1.A, R.G, B1.D, S2.A from Sailors S1, Sailors S2, Boats B1, Boats B2, Reserves R where S1.A = B1.E AND B1.D < R.H AND B2.E = R.G;