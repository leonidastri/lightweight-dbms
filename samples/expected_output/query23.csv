1,2,1,101,1
1,3,1,102,1
3,2,3,101,1
1,2,1,101,2
1,3,1,102,2
3,2,3,101,2
3,2,3,101,3
1,2,1,101,3
1,2,1,101,4
1,3,1,102,3
3,2,3,101,4
1,2,1,101,5
1,3,1,102,4
1,3,1,102,5
3,2,3,101,5
1,2,1,101,6
1,3,1,102,6
3,2,3,101,6